docs/development/examples/nearlib/introduction.md

```diff
@@ -937,16 +942,25 @@ If you saw something unexpected, here's what may have happened ...
 
 **If you saw something** about a `nonce` being invalid it's possible you created the `sender` object with one key and tried to call `sendMoney` with a different key.  Just refresh the page and this will disappear.  A `nonce` is just a number, an integer in this case, and the word `nonce` is short for "`n`umber used `once`". On the NEAR platform we use increasing `nonce` values to avoid the risk of replaying transactions as the network is deciding what work to do next.  Each access key tracks how many times it has been used to sign any executed transactions by incrementing an associated `nonce`.  You can read more about how NEAR uses `nonce` values in the authoritative guide to NEAR internals, the Nearnomicon, [here](http://nomicon.io/Runtime/Scenarios/FinancialTransaction.html?highlight=nonce#creating-a-transaction) and [here](http://nomicon.io/BlockchainLayer/Transactions.html?highlight=nonce#transaction-ordering).
 
-**If you saw nothing** change in your personal account balance, it's likely you're trying to send too little money.  Try adding like 10 zeros and see if things improve :)  No, seriously, check out the box just below about `attoNEAR` and units of measurement on the NEAR platform.
-
 <blockquote class="warning">
 <strong>heads up</strong><br><br>
 
-`NEAR` is the unit of measurement for tokens are measured in NEAR Wallet but `attoNEAR` is the unit of measurement for tokens in `nearlib`.
+`NEAR` is the unit of measurement for tokens are measured in NEAR Wallet but `yoctoNEAR` is the unit of measurement for tokens in `nearlib`.
+
+This makes sense when we're talking about vanishingly small amounts of money like the gas consumed by a transfer (you've been watching that number printed to the console in this lesson, see `console.log(final.transaction.outcome.gas_burnt`) but when we're talking about human numbers then `yoctoNEAR` is a very small number indeed. 
+
+To make this easier, we've included two helper functions for dealing with the the NEAR denominations: 
 
-This makes sense when we're talking about vanishingly small amounts of money like the gas consumed by a transfer (you've been watching that number printed to the console in this lesson, see `console.log(final.transaction.outcome.gas_burnt`) but when we're talking about human numbers then `attoNEAR` is a very small number indeed.
+`parseNearAmount()` to go from NEAR to yoctoNEAR 
 
-That's why any noticeable transfers that change the visible balance of an account in NEAR Wallet have 15 digits or more!
+`formatNearAmount()`to go from yoctoNEAR to NEAR
+
+```
+let amount_in_near, amount_in_yocto
+
+amount_in_yocto = nearlib.utils.format.parseNearAmount(amount_in_near)
+amount_in_near = nearlib.utils.format.formatNearAmount(amount_in_yocto)
+```
```

Comment:
people will start calling our unit of gas "yocto" if we write this.  that's not necessarily a bad thing but we should be deliberate about these choices.  alternatively we could write these two lines as

```
amount_in_y_near = nearlib.utils.format.parseNearAmount(amount_in_near)
amount_in_near = nearlib.utils.format.formatNearAmount(amount_in_y_near)
```